sdks/python/apache_beam/io/avroio.py

```diff
@@ -360,8 +360,10 @@ def num_records(self):
   def records(self):
     decoder = avroio.BinaryDecoder(
         io.BytesIO(self._decompressed_block_bytes))
-    reader = avroio.DatumReader(
-        writers_schema=self._schema, readers_schema=self._schema)
+    # Kwargs writers_schema and readers_schema were renamed to
+    # writer_schema and reader_schema in py3.
+    # Therefore, the kwargs were left out to be both py2 and py3 compatible.
```

Comment:
This comment is relevant for PR review, but I think it's not necessary for readers of the code. I suggest we remove it. If we want to keep the names of positional arguments for readability, we can do:
 
```
writer_schema=self._schema
reader_schema=self._schema
reader = avroio.DatumReader(	    
        writer_schema, reader_schema)
```

Reply:
Ah yes, I was indeed looking for something like this! Thanks.